modules/ROOT/pages/dataweave-type-system.adoc

```diff
@@ -417,3 +417,14 @@ fun read(file: FileData<Binary>) =  "This is a binary file with data " ++ (file.
   text: read({data: "Hello World", name: "myFile.txt"})
 }
 ----
+
+== Reuse Types
```

Comment:
```suggestion
[[reuse_types]]
== Reuse Types
```

Adding that anchor so I can link to it directly from What's New. This way, even if the title of the section ever changes, the link will still work.